src/styles/index.css

```diff
@@ -275,6 +277,8 @@
     --p-color-input-checked-border: var(--p-color-selected);
     --p-color-input-checked-text: var(--p-color-text-default);
 
+    --p-color-input-checkbox-indeterminate-bg-img: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='' viewBox='0 0 16 16'%3e%3cpath stroke='%2370707B' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M4 8h8'/%3e%3c/svg%3e");
```

Comment:
The difference here is having the lightmode use a different stroke color - here it's using `%2370707B` which is a url encoded hex `#70707B` which is our `--p-color-input-text` in light mode. 

effectively doing what the before-my-changes `color: var(--p-color-input-text)` was intending to do.

Reply:
sucks that i can't pull in a variable for this. 


and a little sucks also that this isn't immediately obvious that it's using a color from a variable (like via cmd+F searching for the hex value)

😞